site/_data/sidebar.yaml

```diff
@@ -91,6 +88,14 @@ groups:
   - title: Codebase
     endpoint: Codebase
 
+- title: Advanced
```

Comment:
Separate `Spark Streaming Receiver` and `Apache Storm Adaptor` into their own section `Adapters`